src/main/java/com/redhat/coolstore/service/ShoppingCartOrderProcessor.java

```diff
@@ -1,35 +1,33 @@
 package com.redhat.coolstore.service;
 
 import java.util.logging.Logger;
-import javax.ejb.Stateless;
-import javax.annotation.Resource;
-import javax.inject.Inject;
-import javax.jms.JMSContext;
-import javax.jms.Topic;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.annotation.Inject;
+import io.quarkus.micrometer.runtime.MicrometerEmitter;
 
 import com.redhat.coolstore.model.ShoppingCart;
 import com.redhat.coolstore.utils.Transformers;
 
-@Stateless
+@ApplicationScoped
```

Comment:
If this is stateless, and like above, this should not be application scoped but probably something else